apriltag.c

```diff
@@ -306,7 +312,6 @@ static void quick_decode_codeword(apriltag_family_t *tf, uint64_t rcode,
 
         rcode = rotate90(rcode, tf->nbits);
     }
-
```

Comment:
Same here.. move into a dedicated commit for things that are not related to the removal of `exit()`.